packages/salesforcedx-vscode-core/src/commands/forceSourceRetrieveManifest.ts

```diff
@@ -53,10 +56,23 @@ export async function forceSourceRetrieveManifest(explorerPath: vscode.Uri) {
       return;
     }
   }
+
+  const messages: ConflictDetectionMessages = {
+    warningMessageKey: 'conflict_detect_conflicts_during_retrieve',
```

Comment:
```suggestion
    warningMessageKey: nls.localize('conflict_detect_conflicts_during_retrieve'),
```

Reply:
Disregard - I see that we localize the warningMessageKey in postconditionCheckers.